lib/Optimizer/CodeGen/ConvertToQIR.cpp

```diff
@@ -204,6 +204,43 @@ void cudaq::opt::initializeTypeConversions(LLVMTypeConverter &typeConverter) {
   cudaq::opt::populateCCTypeConversions(&typeConverter);
 }
 
+/// @brief Compute function argument size in bytes and offsets
+/// of all the arguments that satisfy the filter.
+/// Start computing offsets and size from `startingArgIdx`.
+std::pair<std::size_t, std::vector<std::size_t>>
+cudaq::opt::getFunctionArgumentLayout(mlir::ModuleOp module,
+                                      mlir::func::FuncOp func,
+                                      bool filter(mlir::Type arg),
+                                      std::size_t startingArgIdx) {
+  auto arguments = func.getArguments();
+  auto results = func.getResultTypes();
+  auto funcTy = func.getFunctionType();
+  auto bufferTy =
+      cudaq::opt::factory::buildInvokeStructType(funcTy, startingArgIdx);
+  mlir::StringRef dataLayoutSpec = "";
```

Comment:
```suggestion
  StringRef dataLayoutSpec;
```
I believe that creates an empty StringRef.